### PR TITLE
Anjith05 add hometown

### PR DESCRIPTION
**Pull request by anjith05**

### CODE REVIEW

**Comment by anjith05** [comment]
:+1:

**Comment by anjith05** [comment]
@stevencericson what would you like to do in Chicago?

File: Anjith05-Chicago.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Chicago, IL
+
+### Places to visit
+
+- Navy Pier
+- Sears tower
+
```
Comment (anjith05) [inline Anjith05-Chicago.md]: Filed museum

File: Anjith05-Chicago.md [hunk 2]
```diff
@@ -2,4 +2,8 @@
 
 ### Places to visit
 
+- Navy Pier
+- Sears tower
+
 ### Things to do
+- Music Concerts
```
Comment (tonyjobson) [inline Anjith05-Chicago.md]: @anjith05 anyone you recomend catching?

**Comment by tonomoshia** [comment]
What is there to do in Chicago in the winter?

**Comment by anjith05** [comment]
@tonomoshia  you can try ice skating

**Pull request merged**